src/modules/endpoint/sagas/loadCityContent.js

```diff
@@ -73,7 +73,7 @@ export default function * loadCityContent (
 
     const fetchMap = resourceURLFinder.buildFetchMap(
       categoriesMap.toArray().concat(events),
-      (url, path) => buildResourceFilePath(url, path, newCity)
+      (url, path, urlHash) => buildResourceFilePath(url, path, newCity, urlHash)
```

Comment:
I don't get why you added the urlHash here. It's not used in the resourceUrlFinder.

Reply:
It is used in fetchResourceCache. This is visible in NATIVE-328